standalone-metastore/src/main/resources/package.jdo

```diff
@@ -1,1305 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Hopefully you didn't mean to remove this, because that will break a lot of stuff.